src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -170,9 +181,10 @@ const AuthenticatorButton: UISchemaElementComponent<{
             paragraph
             id={`${iconName}-description`}
             sx={{
-              fontSize: '.875rem',
+              fontSize: tokens.TypographySizeSubordinate,
+              color: tokens.TypographyColorSubordinate,
```

Comment:
Per Alex's Figma, do we need to explicitly set `fontWeight: tokens.TypographyWeightBody` or is that already the default here for the description?

Reply:
It's default. But it's better to set explicitly